README.md

```diff
@@ -16,20 +16,26 @@ The [draft site](https://draft-openlibertyio.mybluemix.net/) is built using the
+6. (Temporary for now). Create a branch off of the `development branch` of the [openliberty.io repo](https://github.com/OpenLiberty/openliberty.io). Make the following updates all in the same branch:
+* Until Antora natively supports 'latest' redirecting to the latest version, we have to update what version `latest` in the url redirects to: Update the redirects for `/docs/latest/*`, `/docs/latest/`, and `/docs/latest` in the [redirect file](https://github.com/OpenLiberty/openliberty.io/blob/development/src/main/webapp/WEB-INF/redirects.properties) to redirect to the new released doc version (without a `v` in the url).
+* Update the `./scripts/build_clone_docs.sh vX.0.0.X` command in the [./scripts/build_jekyll_maven.sh](https://github.com/OpenLiberty/openliberty.io/blob/development/scripts/build_jekyll_maven.sh) file to clone the latest doc version to get the javadocs which are not handled completely by Antora yet. 
+* Update the version of `{replace_version page.url 'X.0.0.X' }` in [src/main/content/antora_ui/src/partials/head-info.hbs](https://github.com/OpenLiberty/openliberty.io/blob/master/src/main/content/antora_ui/src/partials/head-info.hbs) so that the canonical meta tag is set to the latest version of the documentation for all versions of the docs. 
+* Update the version of the url `<loc>https://openliberty.io/docs/X.0.0.X/overview.html</loc>` in the [src/main/content/sitemap.xml](https://github.com/OpenLiberty/openliberty.io/blob/master/src/main/content/sitemap.xml).
 
-7. (Temporary for now) Rebuild the [development site](http://dev-openlibertyio.mybluemix.net/) using the [IBM Cloud development pipeline](https://cloud.ibm.com/devops/pipelines/61002d76-1c13-4502-a109-3b794b3f2350?env_id=ibm:yp:us-south) and test that the docs link works and redirects to the newest doc version once the site is done building.
+7. Make a pull request from your branch into the `development` branch. Once the pull request has been reviewed merge it in.
```

Comment:
Do you think this should be part of the previous step so that it's super clear what branch we're talking about?